erdkotlin/src/main/java/com/elrond/erdkotlin/data/transaction/responses/GetTransactionInfoResponse.kt

```diff
@@ -0,0 +1,28 @@
+package com.elrond.erdkotlin.data.transaction.responses
+
+import java.math.BigInteger
+
+internal class GetTransactionInfoResponse(
+    val transaction: TransactionInfoData
+) {
+    data class TransactionInfoData(
+        val type: String,
+        val nonce: Long,
+        val round: Long,
+        val epoch: Long,
+        val value: BigInteger,
+        val sender: String,
```

Comment:
Theoretically, the `senderUsername` and `receiverUsername` are available as well in the response. Some other fields, such as `hyperblockNonce` are also available.

Reply:
Done 👍